### ISSUE TITLE
Ensure alt text for figures in documentation ie vignettes

### ISSUE THREAD

**Issue by joethorley** [bcgov/ssdtools#182]
for impaired vision

**Comment by joethorley** [comment]
From <https://www.w3.org/WAI/tutorials/images/tips/#:~:text=The%20alt%20text%20should%20be,methods%20discussed%20in%20complex%20images.>

- Choosing appropriate text alternatives:
Imagine that you’re reading the web page aloud over the phone to someone who needs to understand the page. This should help you decide what (if any) information or function the images have. If they appear to have no informative value and aren’t links or buttons, it’s probably safe to treat them as [decorative](https://www.w3.org/WAI/tutorials/images/decorative/).

- Prioritize information in text alternative:
Aim to put the most important information at the beginning.

- Length of the text alternative:
The alt text should be the most concise description possible of the image’s purpose. If anything more than a short phrase or sentence is needed, it would be better to use one of the long description methods discussed in [complex images](https://www.w3.org/WAI/tutorials/images/complex/).

- Punctuation within alt attributes:
As for any text, using punctuation in the text alternative makes the information easier to understand. In particular, remember to add space characters in the alt text when there’s no space character between the image and adjacent text, to avoid having words running together when they are read by a screen reader.

- Decorative images
If you use a null (empty) text alternative (alt="") to hide decorative images, make sure that there is no space character in between the quotes. If a space character is present, the image may not be effectively hidden from assistive technologies. For instance, some screen readers will still announce the presence of an image if a space character is put between the quotes.

- Superfluous information in the text alternative:
Usually, there’s no need to include words like “image”, “icon”, or “picture” in the alt text. People who can see will know this already, and screen readers announce the presence of an image. In some situations, it may be important to distinguish between paintings, photographs, or illustrations, etc., but it’s best to avoid the more generic use of the terms.

**Comment by joethorley** [comment]
To do

- [ ] README.Rmd
- [ ] vignettes/additional-technical-details.Rmd
- [ ] vignettes/customizing-plots.Rmd
- [ ] vignettes/distributions.Rmd
- [ ] vignettes/faqs.Rmd
- [ ] vignettes/model-averaging.Rmd
- [ ] vignettes/ssdtools.Rmd

**Comment by joethorley** [comment]
done
